CDP4DiagramEditor/Helpers/UpdateGroupAction.cs

```diff
@@ -0,0 +1,215 @@
+            }
+            set
+            {
+                this.SetValue(UpdateGroupAction.PropertyProperty, (object)value);
+            }
+        }
+
+        /// <summary>
+        ///   <para>Gets or sets a name of the property whose value should be updated. This is a dependency property.</para>
+        /// </summary>
+        /// <value>A name of the property whose value should be updated.</value>
+        public string PropertyName
+        {
+            get
+            {
+                return (string)this.GetValue(UpdateGroupAction.PropertyNameProperty);
+            }
+            set
+            {
+                this.SetValue(UpdateGroupAction.PropertyNameProperty, (object)value);
+            }
+        }
+
+        /// <summary>
+        ///   <para>Gets or sets a new value of an updated property. This is a dependency property.</para>
+        /// </summary>
+        /// <value>A new value of an updated property.</value>
+        public object Value
+        {
+            get
+            {
+                return this.GetValue(UpdateGroupAction.ValueProperty);
+            }
+            set
+            {
+                this.SetValue(UpdateGroupAction.ValueProperty, value);
+            }
+        }
+
+        /// <summary>
+        ///   <para>Gets or sets the data binding for the updated property.</para>
+        /// </summary>
+        /// <value>A <see cref="T:System.Windows.Data.BindingBase" /> object instance that specifies the data binding for the updated property.</value>
+        public BindingBase ValueBinding { get; set; }
+
+        /// <summary>
+        ///   <para>Gets or sets an element whose properties should be updated. This is a dependency property.</para>
+        /// </summary>
+        /// <value>An element whose property should be updated.</value>
+        public object Element
+        {
+            get
+            {
+                return this.GetValue(UpdateGroupAction.ElementProperty);
+            }
+            set
+            {
+                this.SetValue(UpdateGroupAction.ElementProperty, value);
+            }
+        }
+
+        /// <summary>
+        ///   <para>Gets or sets a name of an element whose property value should be updated. This is a dependency property.</para>
+        /// </summary>
+        /// <value>A name of an element whose property value should be updated</value>
+        public string ElementName
+        {
+            get
+            {
+                return (string)this.GetValue(UpdateGroupAction.ElementNameProperty);
+            }
+            set
+            {
+                this.SetValue(UpdateGroupAction.ElementNameProperty, (object)value);
+            }
+        }
+
+        /// <summary>
+        ///   <para>Returns the object being manipulated by the current action.</para>
+        /// </summary>
+        /// <returns>An object that is manipulated by the current action.</returns>
+        public override object GetObjectCore()
```

Comment:
In this and all methods below there is style deficiencies:

- use `var` instead of type
- if statement spacing and usage of { }